### DISCUSSION TITLE
Dependabot detecting ansible-lint fork

### DISCUSSION THREAD

**Discussion by hemberger** [ansible/ansible-lint#2389] [General]
Hi all! I was looking at my Dependabot dependency list today, and I noticed that it linked to a fork of the ansible-lint repository.

This is happening globally. All ~7k projects that depend on ansible-lint are listed here:
https://github.com/Acidburn0zzz/ansible-lint/network/dependents

Whereas the official ansible-lint repo has _no_ listed dependents:
https://github.com/ansible/ansible-lint/network/dependents

Apparently this is because the packages that are detected in the official repo are named `github.com/ansible-community/ansible-lint` and `github.com/ansible/ansible-lint`, whereas the fork is detected as just `ansible-lint`.

(It is important to note that the actual dependency analysis uses the PyPI package, which comes from here; it's just an issue with the linkage that github.com does.)

Is there anything that can be changed in the ansible-lint repo metadata that could improve this situation? Or is this an issue to raise with github and/or dependabot?

Thanks for your time!

**Comment by cognifloyd** [comment]
ansible-community/ansible-lint moved to ansible/ansible-lint, so the ansible-community/ variant is probably listed in other releases, but it should redirect to ansible/ following the move.
